ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/ArrayOrListExpr.cs

```diff
@@ -0,0 +1,10 @@
+﻿using JetBrains.ReSharper.Plugins.FSharp.Psi.Parsing;
+using JetBrains.ReSharper.Psi.Tree;
+
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.Tree
+{
+  internal partial class ArrayOrListExpr
+  {
+    public bool IsArray => FirstChild?.GetTokenType() == FSharpTokenType.LBRACK_BAR;
```

Comment:
It'd probably be better to have separate types for lists and arrays in the psi grammar. The current design comes from FCS.

Reply:
@DedSec256 Thanks for updating the psi!